app/main.py

```diff
@@ -1,6 +1,18 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+
+    cache_storage = {}
+
+    def inner(*args) -> Any:
```

Comment:
- Good to add an annotation from `functools` module
- Also, it should work with `**kwargs`. Create a unique key using args and kwargs 

```suggestion
    @wraps(func)
    def inner(*args, , **kwargs) -> Any:
```